docs/artifactory/transfersettings/help.go

```diff
@@ -0,0 +1,7 @@
+package transfersettings
+
+var Usage = []string{"rt transfer-settings"}
+
+func GetDescription() string {
+	return "Configure Artifactory transfer."
```

Comment:
Configure Artifactory transfer.
-->
Configure the settings for the 'jf rt transfer-files' command